tests/wdio-standard.conf.js

```diff
@@ -0,0 +1,33 @@
+const wdioBaseConfig = require('./wdio.conf');
+const _ = require('lodash');
+const utils = require('./utils');
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+
+// Override specific properties from wdio base config
+const standardConfig = _.merge(wdioBaseConfig.config, {
+  specs: [
+    './tests/e2e/standard-config/**/*.wdio-spec.js'
+  ],
+  
+  onPrepare: async function () {
+    await utils.prepServices({ suite: 'web' });
```

Comment:
It's probably best to update the `prepServices` signature to take a second argument to disable that instead of passing an arbitrary suite name in.